tests/typecheck/managers/test_managers.yml

```diff
@@ -386,9 +386,11 @@
 -   case: override_manager_create1
     main: |
         from myapp.models import MyModel
-        MyModel.objects.create()
     installed_apps:
         - myapp
+    out: |
+        myapp/models:4: error: Return type "MyModel" of "create" incompatible with return type "_T" in supertype "BaseManager"
```

Comment:
This looks like a problem: `BaseManager` defines `create() -> [T: Model]`, and our `ModelManger(BaseManager)` defines it as `create() -> MyModel`. Why is that an error?

Reply:
Hmm, yeah, I can dig a bit deeper into this

Reply:
The error here is exactly the same as you'd get from mypy if you did this:

```python
from typing import TypeVar
from django.db import models

T = TypeVar("T", bound=models.Model)

class MyManager(models.Manager[T]):
    def create(self, **kwargs) -> "MyModel":
        ...

class MyModel(models.Model):
    ...
```

So it's expected based on what the plugin is doing, but I guess it might be surprising?

Reply:
Yes, I now understand. The error is correct, but is very confusing.

Last question: can we somehow make it more clear? Or complete remove it (eventhough it iw correct)?

Reply:
Hmm, I can see what I can do, but I'm not aware of any methods we have available to change mypy's warnings?

At the very least I can add a note about this in the readme. I _might_ also be able to detect when this has happened and omit an additional error maybe, but I'm not sure if that helps much?

Reply:
We can add a `note` saying that it probably should be `class MyManager(models.Manager[MyModel]):`, not `class MyManager(models.Manager[T]):`

How does that sound?

Reply:
Or we can just ignore it for now :)